### PR TITLE
Port performance tests

### PR DESCRIPTION
**Pull request by lubosmj**
Compared to existing performance tests, fixtures are generated in
/usr/local/lib/pulp/lib/python3.7/site-packages/rest_framework/static/fixtures/.
And then available at http://localhost:80/static/fixtures/.

It order to execute these tests, it is required to define CONTENT_HOST
in /etc/pulp/settings.py.

closes #5458
https://pulp.plan.io/issues/5458

### CODE REVIEW

**Comment by bmbouter** [comment]
@lubosmj can you rebase against the latest master to receive https://github.com/pulp/pulp_file/pull/286

**Comment by lubosmj** [comment]
The PR was rebased.

File: pulp_file/tests/performance/pulpperf/reporting.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    """Return overview of tasks."""
+    out = []
+    for t in tasks:
+        out.append(t["_href"])
+        out.append("    name:\t%s" % t["name"])
+        out.append("    created:\t%s" % t["_created"])
+        out.append("    started:\t%s" % t["started_at"])
+        out.append("    finished:\t%s" % t["finished_at"])
+        out.append("    state:\t%s" % t["state"])
+    return "\n".join(out)
```
Comment (fabricio-aguiar) [inline pulp_file/tests/performance/pulpperf/reporting.py]: These reporting methods are great!

**Comment by bmbouter** [comment]
I didn't provide enough feedback in time. Sorry about that @lubosmj 

I ran these locally just now, and I see they need updates for the field renaming. For example I get the error below.

```
======================================================================
ERROR: test_01_sync_repository (pulp_file.tests.performance.test_performance.PerformanceTestCase)
Measure time of synchronization.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vagrant/devel/pulp_file/pulp_file/tests/performance/test_performance.py", line 59, in test_01_sync_repository
    r["repository_href"] = create_repo(r["repository_name"])
  File "/home/vagrant/devel/pulp_file/pulp_file/tests/performance/test_performance.py", line 266, in create_repo
    return interact.post("/pulp/api/v3/repositories/", data={"name": name})["_href"]
KeyError: '_href'
```

**Comment by lubosmj** [comment]
The errors should be resolved now. Note that the fixture generator does not have files-count set to 100000 as stated in the issue description (https://github.com/pulp/pulp_file/pull/283/files#diff-ab155ce0c3818949ed06d7044e5f3a0cR17). For personal testing purposes, it was not appropriate to run such long running tasks, therefore the files-count was set to 100 only. Still, I can update the PR, if you insist.

**Comment by bmbouter** [comment]
@lubosmj yes please set it to `100000`.

Also is it possible to have it print output similar to the format in [this example](https://travis-ci.org/pulp/pulp_rpm/jobs/600836314#L1239). Those tests are in pulp_rpm that have the reporting code. What do you think about this?

**Pull request merged**